treeherder/intermittents_commenter/commenter.py

```diff
@@ -0,0 +1,285 @@
+import json
+import re
+import time
+from collections import Counter
+from datetime import (date,
+                      datetime,
+                      timedelta)
+
+import requests
+from django.conf import settings
+from django.db.models import Count
+from jinja2 import Template
+from requests.exceptions import RequestException
+
+from treeherder.intermittents_commenter.constants import *
```

Comment:
Wildcard imports are generally discouraged in Python, could you list them individually?